apps/mobile/src/ui/ShareQRCodeCard/ShareQRCodeCard.tsx

```diff
@@ -78,99 +78,97 @@ export const ShareQRCodeCard = ({
 
   return (
     <TouchableWithoutFeedback onLongPress={onLongPress}>
-      <>
-        <View
+      <View
+        style={[
+          a.gap_lg,
+          a.align_center,
+          a.flex_1,
+          a.px_lg,
+          {
+            minHeight: 432,
+            borderRadius: 16,
+            alignItems: 'center',
+            width: screenWidth - 32,
+            overflow: 'hidden',
+          },
+        ]}
+      >
+        <LinearGradient
           style={[
-            a.gap_lg,
-            a.align_center,
-            a.flex_1,
-            a.px_lg,
             {
-              minHeight: 432,
+              position: 'absolute',
+              top: 0,
+              left: 0,
+              right: 0,
+              bottom: 0,
+              opacity: 1,
               borderRadius: 16,
-              alignItems: 'center',
-              width: screenWidth - 32,
-              overflow: 'hidden',
             },
```

Comment:
opacity: 1 is default 
```suggestion
            },
            a.absolute,
            a.inset_0,            
```